engine/time/src/main/java/io/deephaven/time/DateTimeUtils.java

```diff
@@ -135,19 +135,20 @@ public class DateTimeUtils {
     public static final long HOUR = 60 * MINUTE;
 
     /**
-     * One day in nanoseconds.
+     * One day in nanoseconds. This is one hour of wall time and does not take into account calendar adjustments.
      */
     public static final long DAY = 24 * HOUR;
 
     /**
-     * One week in nanoseconds.
+     * One week in nanoseconds. This is 7 days of wall time and does not take into account calendar adjustments.
      */
     public static final long WEEK = 7 * DAY;
 
     /**
-     * One year (365 days) in nanoseconds.
+     * One average year in nanoseconds. This is 365.2425 days of wall time and does not take into account calendar
+     * adjustments.
      */
-    public static final long YEAR = 365 * DAY;
+    public static final long YEAR = 31556952000000000L;
```

Comment:
could you define this as `(long) (365.2425 * DAY)`? it's just faster/more clear when looking at the source (especially if you don't have the javadocs available and can't see the comment)

Reply:
The docs and current revision should make this clear.